src/pages/docs/projects/ephemeral-environments/index.md

```diff
@@ -136,7 +154,13 @@ Environments can be filtered by name and by the current state of the environment
 
 ## Updating an existing environment
 
-To update an existing ephemeral environment, create another release that results in the same environment name based on the template. The release will be automatically deployed into the environment.
+### Automatic Deployments
+
+Create another release that results in the same environment name based on the environment name template.
```

Comment:
It's implied that we'll automatically deploy the release, but prob still good to include the explicit text I think to make it super clear what will happen.

```suggestion
Create another release that results in the same environment name based on the environment name template. The release will be automatically deployed into the environment.
```